### ISSUE TITLE
feature: Java Language

### ISSUE THREAD

**Issue by giraffeingreen** [windmill-labs/windmill#4102]
Is there a plan to support Java language?

**Comment by rubenfiszel** [comment]
Yes, one day but not priority right now as we lack customer requests for it. Other languages we intend to support are R and Rust
